### PR TITLE
solved week6-prob1

### LINKED ISSUE #10: 6주차 과제 풀이
- 주제: 문자열 출력하기
---
- [x] PR 올리기
- [ ] 본인 이외 코드 리뷰 남기기
    - [x] 유진 
    - [ ] 동우
    - [x] 지영
- [x] 깃허브 꾸며놓기

### PR DESCRIPTION
**Pull request by hnyoojin**
- #10 

Solved Programmers level0 probs

### CODE REVIEW

File: Yoojin-week6 [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    scanf("%s", s1);
+
+    int len = strlen(s1);
+    for (int i = 0; i < len; i++){
+        printf("%c", s1[i]);
+    }
+    printf("\n");
+    return 0;
+}
+
```
Comment (flydongwoo) [inline Yoojin-week6:18]: Wow! I think that it is a spoiler!
Later, I will write some comment!

Comment (hnyoojin) [reply Yoojin-week6:18]: Cool

File: Yoojin-week6 [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+#include <stdio.h>
+#include <string.h>
+#define LEN_INPUT 1000001
+
+int main(void) {
+    char s1[LEN_INPUT];
+    scanf("%s", s1);
+
+    int len = strlen(s1);
+    for (int i = 0; i < len; i++){
```
Comment (jiyeongggg) [inline Yoojin-week6:12]: You use for loop! 
I couldn't think that way! Cooool~

File: Yoojin-week6 [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+        printf("%c", s1[i]);
+    }
+    printf("\n");
```
Comment (jiyeongggg) [inline Yoojin-week6:15]: Finishing with \n is so kind!

File: Yoojin-week6 (hunk 2 above)
Comment (flydongwoo) [reply Yoojin-week6:12]: @jiyeongggg Why are you so good at English? Is this the effect of 실영초?

File: Yoojin-week6 [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+#include <stdio.h>
+#include <string.h>
+#define LEN_INPUT 10
+
+int main(void) {
+    char s1[LEN_INPUT];
+    scanf("%s", s1);
+
+    int len = strlen(s1);
```
Comment (flydongwoo) [inline Yoojin-week6:66]: I'm so impressive to use strlen function!

File: Yoojin-week6 [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+// 특수문자 출력하기
+
+#include <stdio.h>
+
+int main(void) {
+    printf("!@#$%%^&*(\\'\"<>?:;");
+    return 0;
+}
+
+// 잠시 이것만 수정해서 제 폴더에 머지해볼게유...
```
Comment (flydongwoo) [inline Yoojin-week6]: Cool~ Great job!

File: Yoojin-week6 (hunk 4 above)
Comment (hnyoojin) [reply Yoojin-week6:66]: Yeah I think that's useful func

File: Yoojin-week6 (hunk 2 above)
Comment (hnyoojin) [reply Yoojin-week6:12]: @flydongwoo Already? HAHA

File: Yoojin-week6 (hunk 3 above)
Comment (hnyoojin) [reply Yoojin-week6:15]: Yeah,,, I perfer this way!!

File: Yoojin-week6 (hunk 2 above)
Comment (jiyeongggg) [reply Yoojin-week6:12]: lol

**Pull request merged**